refactor: new CLI commands documentation (#413)
NodeSecure/cli: docs/cli/report.md

## 📝 Command `report`

> The `report` command is used to generate a detailed security report for a repository. This report can include all dependencies and be customized with various options such as theme, title, and format.

Reviewer: ```suggestion
The `report` command generates a detailed security report for a repository in PDF format. The report can include all dependencies and can be customized with various options.
```
